Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.Properties.cs

```diff
@@ -67,7 +67,7 @@ public partial class DropShadowPanel
         /// On platforms not supporting drop shadows, this control has no effect.
         /// </remarks>
         public static bool IsSupported =>
-            (!ControlHelpers.IsRunningInLegacyDesignerMode) && ApiInformation.IsTypePresent("Windows.UI.Composition.DropShadow"); // SDK >= 14393
+            (!DesignTimeHelpers.IsRunningInLegacyDesignerMode) && ApiInformation.IsTypePresent("Windows.UI.Composition.DropShadow"); // SDK >= 14393
```

Comment:
This looks unrelated?

Reply:
I moved the design time helpers you did to the base UI package so I can use them outside of the controls package (makes no sense to implement the same thing multiple times). Had to update all references to it.